.github/workflows/main.yml

```diff
@@ -0,0 +1,73 @@
+name: Python package
+
+on: push
+
+jobs:
+  build:
+
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: [2.7.18]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
```

Comment:
IMHO `strategy.matrix` vobec nepotrebujeme a staci sem explicitne napisat `python-version: 2.7.18`. V nazve stepu cislo python verzie nemusi byt, kedze pouzivame iba jednu.

In general, konfiguraciu chceme mat as simple as possible.